code/modules/recycling/conveyor.dm

```diff
@@ -99,99 +125,135 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 		if(SOUTHWEST)
 			forwards = WEST
 			backwards = NORTH
-	if(verted == -1)
+
+	if(inverted)
+		var/temp = forwards
+		forwards = backwards
+		backwards = temp
+	// We need to do this this way to ensure good functionality on corner belts.
+	// Basically, this allows the conveyor belts that used a flipped belt sprite to
+	// still convey items in the direction of their arrows. It's different from inverted,
+	// so they need to be ran separately.
+	if(flipped)
```

Comment:
why not just flipped || inverted?

Reply:
Because they both need to be ran separately.

Reply:
oh they both need to apply